### ISSUE TITLE
Direkt zur letzten Seite eines Threads springen  

### ISSUE THREAD

**Issue by bambus-58** [theZiz/ods-upgrade#35]
Aktuell kann man in mehrseitigen Thread nur eine  Seite vor oder zurückblättern. Ein weiterer Button um zum Ende des Threads zu springen wäre schön

**Comment by bambus-58** [comment]
Hinweis, hab gesehen, das am unteren Ende der auf der Seite angezeigten Beiträge eine bessere Sprungmöglichkeit bei langen Threads vorhanden ist (.
Von daher würde ich das als Prio 2 ansehen.
Bsp. aus den Thread "Wo bin ich?":
![image](https://user-images.githubusercontent.com/76645785/103176634-b03a9400-4873-11eb-8684-f6563efc6f70.png)

**Comment by DerWafer** [comment]
Ja, die Navigationsmöglichkeiten sind jetzt vielfältiger. Früher konnte man oben und unten jeweils über die Breadcrums (z.B. Forum -> ODS - Gemeinschaft ->Lagerfeuer) navigieren. Das geht jetzt nur noch oben. Diese Buttons, von denen du schreibst gabe es früher auch oben und unten. Die gibt es jetzt nur noch unten. Dafür gibt es oben die Navigation über die Seitenauswahl: Auf mobilen Geräten fehlt das aber auf dem PC ist das eine graue Zeile mit einem Button "Antworten" und eben der Seitennavigation. Da wird dir angezeigt, auf welcher Seite von wie vielen Seiten du bist. Und das erste Feld ist editierbar. Da kannst du jede beliebige Seitennummer eintragen. Auch die letzte oder jede beliebige Zahl darüber. Damit kommst du auch auf die letzte Seite. Dann folgen noch 2 Buttons mit "Vorherige Seite" und "Nächste Seite" - halt nur mit Pfeilen drauf. Inhaltlich in etwa das Gleiche wie diese großen Buttons unten, die du hier zeigst.
Ob das in Summe damit besser geworden ist, muss jeder für sich beurteilen. Ich denke hier spielen auch Themen mit rein, die was mit der Darstellbarkeit auf mobilen Devices zu tun haben. Solchen Überlegungen unterlag das alte Forum nicht. Und wenn wir hier was dazubauen, dann muss es rund zu den anderen Dingen passen und mobil auch Sinn machen. Daher sehe ich persönlich hier eher weniger Aktionsbedarf, da es die von dir angesprochene Navigationsmöglichkeit oben und unten auf der Seite gibt.
